tests/Integration/Infrastructure/Persistence/MySQLAssetRepositoryTest.php

```
@@ -0,0 +1,784 @@
+                new AssetId((string) $persistedAsset->getId()),
+                new UploadId((string) $persistedAsset->getUploadId()),
+                new AccountId((string) $persistedAsset->getAccountId()),
+                $persistedAsset->getFileName(),
+                $persistedAsset->getMimeType(),
+                AssetStatus::FAILED,
+                $this->persistedState(
+                    '2026-04-01 13:15:00.000000',
+                    1,
+                    '2026-04-01 13:18:00.000000',
+                ),
+            );
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\App\Domain\Asset\Asset' in method 'itRejectsAStaleSaveWhenTheCompareAndSwapUpdateLosesARace'. (undefined)

(StaticAccess)